drivers/at24cxxx/at24cxxx.c

```diff
@@ -0,0 +1,366 @@
+/**
+ * @ingroup     drivers_at24cxxx
+ * @{
+ *
+ * @file
+ * @brief       Device driver implementation for the AT24C128 / AT24C256 EEPROM units.
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <errno.h>
+#include <string.h>
+
+#define ENABLE_DEBUG  (0)
+#include "debug.h"
+#include "assert.h"
+#include "xtimer.h"
+
+#include "include/at24cxxx_params.h"
+#include "include/at24cxxx_defines.h"
+#include "at24cxxx.h"
+
+/*
+    page 0: 0-63
+    page 1: 64-127
+    page 2: 128-191
+    page 3: 192-255
+    page 4: 256-319
+    page 5: 320-383
+    page 6: 384-447
+    page 7: 448-511
+    page 8: 512-575
+    ...
+ */
+static inline void _position_to_address_words(uint8_t addr[AT24CXXX_ADDR_LEN],
```

Comment:
Isn't this just `htons()`?

Reply:
Yes, I could have used `htons()`. With the `i2c_*_reg()` functions, it should be obsolete anyways, because these actually call `htons()`.